providers/curation/github.js

```diff
@@ -131,6 +132,53 @@ class GitHubCurationService {
     return revision ? extend(true, {}, summarized, revision) : summarized;
   }
 
+  async getContent({ref, path}) {
+    const {owner, repo} = this.options;
+    const github = Github.getClient(this.options);
+    try {
+      const response = await github.repos.getContent({owner, repo, ref, path});
+      return base64.decode(response.data.content);
+    } catch (error) {
+      // @todo add logger
```

Comment:
Great call out. Need to figure the right error handling strategy across the board. Interestingly for some of these REST requests, they only throw and error if there is an error in the operation (e.g., the network is down). Getting a 4** or 5** is still success from the request point of view. So there needs to be "error" handling in the non-error cases as well. Again, lets figure a general strategy and apply.

Reply:
Yeah, let's keep it consistent across the app. Not sure what you guys have used in the past for this. Once we sort that out I'll update.